py/desisim/targets.py

```diff
@@ -279,19 +282,28 @@ def get_targets(nspec, program, tileid=None, seed=None, specmin=0):
         elif objtype == 'ELG':
             from desisim.templates import ELG
             elg = ELG(wave=wave)
-            simflux, wave1, meta1 = elg.make_templates(nmodel=nobj, seed=seed)
+            if specify_targets is None:
+                simflux, wave1, meta1 = elg.make_templates(nmodel=nobj, seed=seed)
+            else:
+                simflux, wave1, meta1 = elg.make_templates(nmodel=nobj, seed=seed, **specify_targets)
             fibermap['DESI_TARGET'][ii] = desi_mask.ELG
 
         elif objtype == 'LRG':
             from desisim.templates import LRG
             lrg = LRG(wave=wave)
-            simflux, wave1, meta1 = lrg.make_templates(nmodel=nobj, seed=seed)
+            if specify_targets is None:
+                simflux, wave1, meta1 = lrg.make_templates(nmodel=nobj, seed=seed)
+            else:
+                simflux, wave1, meta1 = lrg.make_templates(nmodel=nobj, seed=seed, **specify_targets)
             fibermap['DESI_TARGET'][ii] = desi_mask.LRG
 
         elif objtype == 'BGS':
             from desisim.templates import BGS
             bgs = BGS(wave=wave)
-            simflux, wave1, meta1 = bgs.make_templates(nmodel=nobj, seed=seed)
+            if specify_targets is None:
+                simflux, wave1, meta1 = bgs.make_templates(nmodel=nobj, seed=seed)
+            else:
+                simflux, wave1, meta1 = bgs.make_templates(nmodel=nobj, seed=seed), **specify_targets
```

Comment:
Typo in where the parentheses close.  Please run "python setup.py test" locally before pushing to github to catch problems like this before Travis does.  Tests will run faster if you get the desisim-testdata first:
```
git clone https://github.com/desihub/desisim-testdata
source desisim-testdata/setup-testdata.sh
```